weaknesses/MASVS-PRIVACY/MASWE-0109.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secondary purpose of sensitive data use
+id: MASWE-0109
+alias: sensitive-data-secondary-purpose
+platform: ["android", "ios"]
+profiles: ["P"]
+mappings:
+  masvs-v1: 
+  masvs-v2: [MASVS-PRIVACY-1, MASVS-PRIVACY-2, MASVS-PRIVACY-4]
+  cwe: [359]
+status: new
+---
+
+## Overview
+Sensitive data might be used for secondary purposes or beyond the initial scope without notice or consent from users, while a user consent is obtained for the collection of such data for the initial purpose.
+
+## Modes of Introduction
+
+This risk can be introduced in various scenarios, including:
+
+- Use of user data collected for secondary purpose without user consent or control.
+- Continuous access to user data without user controls.
+
+
+## Impact
+
+The impact of exposing sensitive data in network traffic includes:
```

Comment:
This feels like a copy-paste mistake? This weakness isn't related to this.

Reply:
Updated.